docs/concepts.rst

```diff
@@ -120,22 +117,20 @@ In fact, this is how Channels works. The interface servers transform connections
 from the outside world (HTTP, WebSockets, etc.) into messages on channels,
 and then you write workers to handle these messages.
 
-This may seem like it's still not very well designed to handle push-style
-code - where you use HTTP2's server-sent events or a WebSocket to notify
-clients of changes in real time (messages in a chat, perhaps, or live updates
-in an admin as another user edits something).
-
 However, the key here is that you can run code (and so send on channels) in
 response to any event - and that includes ones you create. You can trigger
 on model saves, on other incoming messages, or from code paths inside views
-and forms.
+and forms. That approach comes in handy for push-style
+code - where you use HTTP2's server-sent events or a WebSocket to notify
```

Comment:
Ah of course, you're right. But I just moved the sentence around, because I found it easier to understand that way. Did both of us mean [HTML5 server-side events](http://www.w3schools.com/html/html5_serversentevents.asp) instead of HTTP2?